### ISSUE TITLE
Sök pengar från ufs.se

### ISSUE THREAD

**Issue by Nyholm** [phpse/motioner#1]
Som @etu tidigare har nämt så borde vi söka pengar av ufs.se för att driva föreningen. Det vi eventuellt behöver är att ha lite mer information av fåra användare. Typ address. 

Pengarna ska se till att betala meetup.com och eventuellt talare.

**Comment by magnusnordlander** [comment]
Vi har väl typ inga medlemmar mellan 7-25?

**Comment by etu** [comment]
Äh, det finns flera föreningar inom UFS som knappt har några unga medlemmar.

I lokalen som UFS har så är snittåldern på besökarna ändå typ 25-35.

Så sure, bidragen när man inte har många unga medlemmar är... begränsade så
att säga. Men man har ju fortfarande grundbidraget (som kan täcka meetup om
inte annat) plus att man kan söka projektbidrag och liknande.

**Comment by magnusnordlander** [comment]
Yes, ja, men då är det väl definitivt värt att överväga/diskutera. Sedan får man ju komma ihåg att 750 kr är mindre än vad vi drar in i anmälningsavgift på en vanlig meetup, men det finns ju andra fördelar också.
